### PR TITLE
Contact us Send button fixed

### PR DESCRIPTION
**Pull request by deepakpathania-zz**
Alignment in mobile version fixed.

### CODE REVIEW

**Comment by CuriousLearner** [comment]
Hi @deepakpathania Thanks for you contribution.

Please consider working on develop branch from the next time onwards :) It's generally a good habit.

cc @Slayslot Check this please, and merge if it is good.

**Comment by deepakpathania-zz** [comment]
@CuriousLearner Yeah, I will work on the develop branch from now on. Thanks for pointing it out.

File: src/client/css/style.css [hunk 1]
```diff
@@ -1703,7 +1703,7 @@ section#contact_form .text-danger {
   font-weight: 700;
   width: 200px;
   height: 50px;
-  margin: 0 0 0 22px;
+  /*margin: 0 0 0 20px;*/
```
Comment (handasolo) [inline src/client/css/style.css:1706]: Commenting this line seems to make it deviate from the center on lg screens. But I would hate for it to work through adjusting margins magic numbers via media queries.

**Comment by deepakpathania-zz** [comment]
@Slayslot It deviated from the original on lg screens but still looked fine on it and worked well for xs screens, that's why I commented it out. Hard coding margins didn't seem like a great way to me. Do you want me to put it back and try something else ?

File: src/client/css/style.css (hunk 1 above)
Comment (handasolo) [inline src/client/css/style.css:1706]: Take a look at how I make buttons center in the events section of the website. Maybe you find something there.

**Comment by deepakpathania-zz** [comment]
@Slayslot You have used the usual center-block there like you have used here, the difference being the hard coded margin that you have provided in the send-button class, which persists in the smaller screen sizes, giving the shifted look. Using a media query don't seem like a great workaround to me.

**Comment by handasolo** [comment]
@deepakpathania No, I don't want the hard coded margins. It'll be good if we get rid of hard coded magic numbers all over the website. Yeah, I'm well aware of what was causing the deviation on smaller screens. What I can't understand is if the buttons on events section are getting centered on all screens with the `center-block` class, then why do we have a different result here.

**Comment by deepakpathania-zz** [comment]
@Slayslot my work around seem to work fine with a tad bit deviation in lg screen sizes.   I will definitely still try and see what's going wrong by looking at the code properly.

**Comment by handasolo** [comment]
Do take a look @deepakpathania

**Pull request closed without merging**